packages/turf-collect/package.json

```diff
@@ -1,36 +1,35 @@
 {
-  "name": "@turf/collect",
+  "name": "@turf/turf-collect",
   "version": "3.7.3",
   "description": "turf aggregate module",
   "main": "index.js",
   "scripts": {
-    "test": "tape test.js"
+    "test": "node test.js"
```

Comment:
FWIW many of the existing modules say 'node test.js'. Executing via `node test.js` still runs the tests using tape just fine... The only benefit I can see of using it in the `scripts` is that it allows you to [test multiple files using globs ](https://github.com/substack/tape#usage), eg `tape tests/*.js`